### ISSUE TITLE
Transform error should not kill build

### ISSUE THREAD

**Issue by mattdesl** [sheetify/sheetify#48]
Getting this when using an invalid filename to sheetify function:

```
assert.js:89
  throw new assert.AssertionError({
  ^
AssertionError: filename must be a string
    at parseCss (/projects/mattdesl.github.com/node_modules/sheetify/index.js:31:10)
    at sheetify (/projects/mattdesl.github.com/node_modules/sheetify/index.js:23:17)
    at nr (/projects/mattdesl.github.com/node_modules/browserify/node_modules/module-deps/index.js:297:23)
    at /projects/mattdesl.github.com/node_modules/browserify/node_modules/resolve/lib/async.js:44:21
    at ondir (/projects/mattdesl.github.com/node_modules/browserify/node_modules/resolve/lib/async.js:187:31)
    at onex (/projects/mattdesl.github.com/node_modules/browserify/node_modules/resolve/lib/async.js:93:22)
    at /projects/mattdesl.github.com/node_modules/browserify/node_modules/resolve/lib/async.js:24:18
    at FSReqWrap.oncomplete (fs.js:82:15)
```

And it kills budo and other tools. Instead, might be better for the transform to emit an `error` event so budo and similar tools can catch it and display this error in the browser.

**Comment by mattdesl** [comment]
Hmm, I think maybe this issue is a bit different. I get the error even when my path is correct

**Comment by mattdesl** [comment]
D'oh! I'm a tool. I was using `-t sheetify` instead of `-t sheetify/transform`

**Comment by mattdesl** [comment]
I've noticed some other errors like #49 are killing the build, so I will re-open this issue since it's mostly about not killing the build on errors. :smile:

**Comment by yoshuawuyts** [comment]
Ah yes, you're fully right I'll make sure #49 emits an error (let's see if it can be tested properly ^^) Thanks for raising!

**Comment by yoshuawuyts** [comment]
Cool, so I've created patch for this that should at least catch this particular error - I feel some refactoring might be in place to make this more solid. I'll upload it once I get home because I'm silly and don't have a VPN that can circumvent cafe firewall 😕

**Comment by yoshuawuyts** [comment]
Should be resolved in #52 (mostly, I hope we're not accidentally throwing for more runtime errors - hah) Thanks!
